<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListView</class>
 <widget class="QWidget" name="ListView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>List View Model</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_1">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="spacing">
      <number>11</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="insert_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/plus.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delete_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/minus.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kitchensink.qrc"/>
 </resources>
 <connections/>
</ui>
